build/scripts/PreCompileApp.ps1

```diff
@@ -55,6 +55,11 @@ if($appType -eq 'app')
                 }
             }
 
+            if($appBuildMode -eq 'Default' -and (Get-StrictModeBranchStatus)) {
+                # StrictMode tag is present on the branch so we build in StrictMode
+                $appBuildMode = "StrictMode"
```

Comment:
Aren't both checks needed?

E.g. 24.1 is tested against latest released 24.1, as well as 24.0.
Strict mode shouldn't replace the _normal_ breaking changes checks.

Reply:
You're right 😄 AL-Go will need to support another build mode I think. I'll look into it

Reply:
Added this to AL-Go - Should be deployed to preview soon